server/src/main/scala/za/co/absa/atum/server/api/v2/http/Endpoints.scala

```diff
@@ -101,27 +101,28 @@ object Endpoints extends BaseEndpoints {
   }
 
   val getPartitioningCheckpointsEndpoint
-    : PublicEndpoint[(Long, Option[Int], Option[Long], Option[String]), ErrorResponse, PaginatedResponse[
+    : PublicEndpoint[(Long, Int, Long, Option[String]), ErrorResponse, PaginatedResponse[
       CheckpointV2DTO
     ], Any] = {
     apiV2.get
       .in(V2Paths.Partitionings / path[Long]("partitioningId") / V2Paths.Checkpoints)
-      .in(query[Option[Int]]("limit").default(Some(10)).validateOption(Validator.inRange(1, 1000)))
-      .in(query[Option[Long]]("offset").default(Some(0L)).validateOption(Validator.min(0L)))
+      // Tapir's .default works with non-optional types only
```

Comment:
Thanks for the comment, ok I liked when it was explicitly said in the type (= `Option`), but if this is how Tapir works, then it is the way forward. Thanks for this!